client/my-sites/checkout/checkout-thank-you/pending.jsx

```diff
@@ -97,7 +97,22 @@ class CheckoutPending extends PureComponent {
 	}
 
 	render() {
-		const { orderId, siteSlug, translate } = this.props;
+		const { orderId, siteSlug, translate, transaction } = this.props;
+
+		if ( transaction && ORDER_TRANSACTION_STATUS.ASYNC_PENDING === transaction.processingStatus ) {
+			return (
+				<Main className="checkout-thank-you__pending">
+					<EmptyContent
+						illustration={ '/calypso/images/illustrations/illustration-shopping-bags.svg' }
+						illustrationWidth={ 500 }
+						title={ translate( 'Payment pending…' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **1** time:
`translate( 'Payment pending' )` *ES Score: 20*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/payment-type-sofort)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).